src/ContactModels/src/ContinuousContactModel.cpp

```diff
@@ -0,0 +1,239 @@
+/**
+ * @file ContinuousContactModel.cpp
+ * @authors Giulio Romualdi
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <iDynTree/Core/EigenHelpers.h>
+
+#include <BipedalLocomotionControllers/ContactModels/ContinuousContactModel.h>
+
+using namespace BipedalLocomotionControllers::ContactModels;
+
+void ContinuousContactModel::setState(const std::unordered_map<std::string, std::any>& state)
+{
+    if (!getVariable(state, "twist", m_twist))
+        throw std::runtime_error("[ContinuousContactModel::setState] Unable to get the variable "
+                                 "named twist");
+
+    if (!getVariable(state, "frame_transform", m_frameTransform))
+        throw std::runtime_error("[ContinuousContactModel::setState] Unable to get the variable "
+                                 "named frame_transform");
+
+    if (!getVariable(state, "null_force_transform", m_nullForceTransform))
+        throw std::runtime_error("[ContinuousContactModel::setState] Unable to get the variable "
+                                 "named null_force_transform");
+
+    // the parameters has been update the previous quantities has to be evaluated again
+    m_isContactWrenchComputed = false;
+    m_isControlMatrixComputed = false;
+    m_isAutonomusDynamicsComputed = false;
```

Comment:
To be honest I don't like too much that child classes have to set a parameter that is used in the implementation of the base class. IMHO, the setting of these parameters to false should be handled by the base class or through a method in the base class, like ``invalidateBuffers()`` or similar.

Reply:
I agree!

Reply:
I implemented the solution you suggested in https://github.com/dic-iit/bipedal-locomotion-controllers/issues/33#issuecomment-604329915